source/user-manual/capabilities/file-integrity/advanced-settings.rst

```diff
@@ -67,6 +67,33 @@ You need to install the audit daemon if you don’t have it already installed on
          # cp /usr/sbin/audisp-af_unix /sbin/audisp-af_unix
          # rc-service auditd restart
 
+.. note::
+
+   On certain systems, auditd is configured with a default rule that causes all audit rule processing to be skipped. This disables any whodata-related configuration of Wazuh.
+   
+   Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT
+
+   Check auditd rules:
+  
+   .. code-block::
+
+      auditctl -l
+      -a never,task
```

Comment:
What is `-a never,task` here? Is this the output of the `auditctl -l` command? Is this the real output or an output when piping the grep command? We could use a separate output code-block to show the output. Also, commands in code-blocks need to include the prompt symbol. `code-block` directives need to specify a language for syntax highlighting, for example for Linux command lines we use `.. code-block:: console`.

Reply:
Done